src/app/docs/alert/alert-docs.component.html

```diff
@@ -4,10 +4,269 @@
+              imageSource="~/assets/img/guidelines/alert/alert-type-4.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Warning alert – for messages the user should know about and perhaps act on, but that do not constitute an error state or indicate that something is wrong.
+              </p>
+            </td>
+          </tr>
+        </table>
+      </sky-docs-design-guideline>
+        
+
+      <sky-docs-design-guideline
+        heading="Close button"
+      >
+        <p>
+          If the user should be able to dismiss the alert from the page after reading, you can include an optional close button.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-option-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <p>
+          Do not make an alert dismissible if it is important for users see every time or if it is triggered by underlying data.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Text inside an alert can contain hyperlinks. For accessibility reasons, these links will not be styled blue as other links in the system are.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-usage-2.png"
+          caption="Do use alerts when a system state requires a call to action."
+          captionType="success"
+        >
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Layout"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          When present, an alert should always be placed at the top of the page, above the page header.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-layout-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        </sky-docs-design-guideline>
+        <sky-docs-design-guideline>
+        <p>
+          Be careful of situations where multiple alerts may stack up; they quickly lose their value and become cluttered if several are present. Think about alternate methods of presenting the information or consolidating messages into one alert.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-layout-2.png"
+          caption="Don't use alerts in a way that allows them to accumulate."
+          captionType="danger"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-layout-3.png"
```

Comment:
In this image, "pm" should be "p.m." and we should have a period after "mail."

Also, we should either delete the semicolon, or we should lowercase the second "do."